src/vehicle.cpp

```diff
@@ -3339,9 +3327,8 @@ float vehicle::wheels_area (int *const cnt) const
 float vehicle::k_friction() const
 {
     // calculate safe speed reduction due to wheel friction
-    float fr0 = 1000.0;
-    float kf = ( fr0 / (fr0 + wheels_area()) );
-    return kf;
+    constexpr float fr0 = 9000.0;
```

Comment:
Might be best to document is as unknown magical value - at least that makes it clear future removal would be acceptable